docs/ARCHITECTURE.md

```diff
@@ -74,7 +74,11 @@ flowchart LR;
     import --> backend
 ```
 
-The delivered formats are loaded into objects. In general, we expect 2-dimensional data.
+**1st step - Extracting** Delivered formats are loaded into objects. In general we expect 2 dimensional data - the data set.
+
+**2nd step - Loading** A dataset contains multiple tupel. A tupel in turn consists of multiple attributes. Imports my support separate headlines. In case of csv the first row could be a headline. Usually each csv-row is a tupel/data point.  
```

Comment:
```suggestion
**2nd step - Loading** A dataset contains multiple tupel. A tupel in turn consists of multiple attributes. Imports may support separate headlines. In case of csv the first row could be a headline. Usually each csv-row is a tupel/data point.  
```